model/patch/github.go

```diff
@@ -32,66 +33,97 @@ type Intent interface {
 	GetType() string
 }
 
-// githubIntent represents an intent to create a patch build as a result of a
+// GithubIntent represents an intent to create a patch build as a result of a
 // PullRequestEvent webhook. These intents are processed asynchronously by an
 // amboy queue.
-type githubIntent struct {
+type GithubIntent struct {
 	// ID is created by the driver and has no special meaning to the application.
-	ID bson.ObjectId `bson:"_id"`
+	Id bson.ObjectId `bson:"_id"`
```

Comment:
I'd leave this as ID

Reply:
Id is consistent with the rest of the models. Do you still want it changed back to ID?